gradle.properties

```diff
@@ -69,7 +69,7 @@ jjwtVersion=0.9.1
 bouncyCastleVersion=1.69
 
 # Version and base tags can be overridden at build time
-pravegaVersion=0.10.0-SNAPSHOT
+pravegaVersion=0.10.1-SNAPSHOT
```

Comment:
@pbelgundi We first need to set this to `0.10.0`, do the release, then change it to `0.10.1` after the release is done. That way the build artifacts point to a commit in GH that has the correct version set.

Reply:
Ok done